### DISCUSSION TITLE
any solution for arbitrary variants multiple style classes?

### DISCUSSION THREAD

**Discussion by hansonfang** [tailwindlabs/tailwindcss#10530] [Help]
As for https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants there is a demo of `[&:nth-child(3)]:underline` which just make the 3rd item `underline`. I just want to know what if I would like it also turning red?
I've tried `[&:nth-child(3)]:underline:text-red-500`、 `[&:nth-child(3)]:(underline text-red-500)` etc. but it doesn't work. any one got solution?

**Comment by wongjn** [comment]
Repeat it for each class like `[&:nth-child(3)]:underline [&:nth-child(3)]:text-red-500`.

**Reply by hansonfang** [reply]
It doesn't look like there is any other way.

**Reply by CanRau** [reply]
Other than switching to UnoCSS ^^
